src/screens/ContactScreen/index.js

```diff
@@ -4,7 +4,7 @@ import {DataProvider, LayoutProvider, RecyclerListView} from 'recyclerlistview';
 import {Dimensions, RefreshControl, SafeAreaView, StatusBar, StyleSheet, View} from 'react-native';
 /* eslint-disable no-param-reassign */
 import {debounce} from 'lodash';
-
+import {useRoute} from '@react-navigation/core';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useRoute` hook from `@react-navigation/core` is imported but there's no check if the route parameters are defined before accessing them. This could potentially lead to a runtime error if the parameters are undefined.
```diff
- const {from: sourceScreen} = route.params;
+ const {from: sourceScreen} = route.params || {};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->